libobs-d3d11/d3d11-subsystem.cpp

```diff
@@ -302,6 +302,10 @@ void gs_device::InitCompiler()
 				module, "D3DDisassemble");
 #endif
 			if (d3dCompile && d3dCreateBlob) {
+				if (ver < 47)
```

Comment:
I don't think this can be true because OBS needs Windows 10, and this is guaranteed in Windows 8.1.

Reply:
Although I think this d3dcompiler_47.dll that ships with Windows is technically for Store/UWP applications, and desktop applications are expected to include the redist DLL from the Windows SDK.

Reply:
The idea is to catch potentially broken DLLs installed by other apps (there's a case where some broken installer overwrites the system32 D3DCompiler with an x86 version). Ideally it will never be true, but you never know what's out there.

Reply:
That seems like more of a reason to include the redist DLL.

Reply:
I can just remove this commit for nwo if it needs further discussion, just included it as it was part of the branch.